sql/catalyst/src/test/scala/org/apache/spark/sql/RowTest.scala

```diff
@@ -121,10 +121,6 @@ class RowTest extends FunSpec with Matchers {
       externalRow should be theSameInstanceAs externalRow.copy()
     }
 
-    it("copy should return same ref for internal rows") {
```

Comment:
ah got it, but it's not true for internal row right? It can be mutable so it's safe to remove this test.